tests/Component.Tests/project.json

```diff
@@ -9,6 +9,7 @@
 		"Components": "1.0.0-*",
 		"FluentAssertions": "4.0.0",
 		"Microsoft.Framework.DependencyInjection": "1.0.0-beta8",
+		"TestAttributes": "2.0.0-rc1",
 		"xunit": "2.1.0",
```

Comment:
Wondering whether we need to explicitly state the xunit dependencies? I wouldn't have said so.